docs/conf.py

```diff
@@ -59,7 +59,8 @@
 add_module_names = False
 modindex_common_prefix = ["circuit_knitting."]
 
-html_theme = "qiskit"
+html_theme = "qiskit-ecosystem"
+html_title = f"{project} {release}"
```

Comment:
This default to `f"{project} {release} documentation"`, which is redundant and noisy. The value gets used for the left sidebar & page title in the browser.

Reply:
The only possible comment I can think of is that `f"{project}"` _might_ be sufficient.  But I don't see anything wrong with including the version, too.

Reply:
Yeah, for the Sphinx Docs Guide I went with just `project` since versioning doesn't matter there. But I figure most projects do care about versioning so should include it.